core/src/main/resources-filtered/org/pentaho/platform/dataaccess/datasource/wizard/public/modeler.properties

```diff
@@ -142,6 +142,14 @@ validation.level.MISSING_LAT=Level must have a Latitude member property
 validation.level.MISSING_LON=Level must have a Longitude member property
 default_aggregation=Default Aggregation
 
+# Model Tree alternative text
+modeler.alternative_text.model=Model
+modeler.alternative_text.dimension=Dimension
+modeler.alternative_text.category=Category
```

Comment:
for deciding the text, based it off what the image looked like and/or the column it was in the data source modeler screen.